src/app/network/network-list/network-card/network-card.component.html

```diff
@@ -0,0 +1,39 @@
+<div class="mb-4 w-full lg:flex c-network-card" data-testid="c-network-card">
+	<div
+		class="w-full border border-gray-400 bg-white rounded-lg leading-normal"
+	>
+		<div class="flex items-center justify-between">
+			<div class="ion-padding flex">
+				<div
+					class="rounder-full network-card--img"
```

Comment:
You can also use the img tag instead of adding it as a background.

```suggestion
					class="rounded-full network-card--img"
```